docs/source/contributors/workflow.rst

```diff
@@ -94,3 +94,42 @@ the PR that this being backported separately.
 
 If you encounter problems with cherry picking into release branch please
 don't hesitate to speak to an experienced member of the FitBenchmarking team.
+
+
+================
+Create a release
+================
+In order to create a new release for FitBenchmarking, there are a few manual steps.
+These have been streamlined as much as possible.
+
+First checkout the branch to create the release from.
```

Comment:
```suggestion
First checkout the branch to create the release from.  Releases should only be made from a `release-x-x` branch, not a development branch or master.
```

Reply:
I can't think of a reason why we'd not do this -- can you?  We should probably enforce this in the script.

Reply:
I agree, might as well check.